docs/azure/ai/get-started-app-chat-scaling-with-azure-container-apps.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Scale Azure OpenAI for .NET chat sample using RAG
+description: Learn how to add load balancing to your application to extend the chat app beyond the Azure OpenAI token and model quota limits.
+ms.date: 03/06/2024
+ms.topic: get-started
+ms.custom: devx-track-java, devx-track-java-ai
+# CustomerIntent: As a .NET developer new to Azure OpenAI, I want to scale my OpenAI capacity to avoid rate limit errors.
+---
+
+# Scale Azure OpenAI for .NET chat using RAG with Azure Container Apps
+
+[!INCLUDE [aca-load-balancer-intro](~/azure-dev-docs-pr/articles/intro/includes//scaling-load-balancer-introduction-azure-container-apps.md)]
```

Comment:
The file isn't in the PR, but can we update the include to link to "the chat app" - if someone lands on this page directly they might not know what that's referring to.